client/pages/trip.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import * as Icons from '../components/svg';
+
+export default class TripSummary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentTrip: null,
+      todoList: [],
+      travelersList: [],
+      transportationList: [],
+      accommodationList: [],
+      activitiesList: [],
+      placesList: [],
+      packingList: []
```

Comment:
I'm not too sure that these are for.

Reply:
For this part, I kind of... thought ahead. It's not needed now.